frontend/survey-project/src/app/app-routing.module.ts

```diff
@@ -33,7 +33,7 @@ const routes: Routes = [
   {
     path: RouteNames.SurveyRoute,
     loadChildren: () => import('./components/pages/survey/survey.module').then(m => m.SurveyModule),
-    canActivate: [authGuard]
+    //canActivate: [authGuard]
```

Comment:
If authentication checks are inside the survey routing module, maybe remove this comment.